public/index.html

```diff
@@ -7,7 +7,7 @@
     <meta name="theme-color" content="#000000" />
     <meta
       name="description"
-      content="Web site created using create-react-app"
+      content="Rocket fuel your progamming skills with Windsor Code Camp"
```

Comment:
Dont you want to add Python Programming because at the moment, that's all we have